popup.css

```diff
@@ -0,0 +1,139 @@
+*{
+    box-sizing: border-box;
+}
+body {
+    background-color: white;
+    min-width: 230px;
+    margin:10% auto;
+}
+div{
+    width:90%;
+    margin: 5px auto;
+    text-align: center;
+}
+footer{
+    display:block;
+    height: 30px;
+    text-align: center;
+}
+h1{
+    font-weight: bold; 
+    text-align: center;
+    font-size: 25px;
+    margin:0;
+    color:#536DFE;
+    display: block;
+}
+h4{
+    text-align: center;
+    margin:0;
+    padding:0;
+}
+label{
+    text-align: center;
+    margin:auto;
+    display: inline-block;
+}
+input{
+    -webkit-transition: 0.3s;
+    transition: 0.3s;
```

Comment:
What is this doing? It seems like it's missing a property based on the way it's described [here](https://www.w3schools.com/css/css3_transitions.asp).